front/app/services/singleSignOn.ts

```diff
@@ -38,5 +39,9 @@ export const handleOnSSOClick = (
     sso_verification_type: verificationContext?.type,
   };
   const urlSearchParams = stringify(omitBy(ssoParams, isNil));
-  window.location.href = `${AUTH_PATH}/${provider}?${urlSearchParams}`;
+  if (provider === 'vienna') {
```

Comment:
Meh, I'm unhappy about this conditional here. The issue is that we the URL in the back-end for the `vienna` SSO is `/auth/saml`, not `/auth/vienna` - and I couldn't find a way of easily changing it.